src/components/PostDetails/PostDetails.jsx

```diff
@@ -1,45 +1,116 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+import PropTypes from 'prop-types';
 import { NewCommentForm } from '../NewCommentForm';
 import './PostDetails.scss';
 
-export const PostDetails = () => (
-  <div className="PostDetails">
-    <h2>Post details:</h2>
-
-    <section className="PostDetails__post">
-      <p>sunt aut facere repellat provident occaecati excepturi optio</p>
-    </section>
-
-    <section className="PostDetails__comments">
-      <button type="button" className="button">Hide 2 comments</button>
-
-      <ul className="PostDetails__list">
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>My first comment</p>
-        </li>
-
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>sad sds dfsadf asdf asdf</p>
-        </li>
-      </ul>
-    </section>
-
-    <section>
-      <div className="PostDetails__form-wrapper">
-        <NewCommentForm />
-      </div>
-    </section>
-  </div>
-);
+import { getPostDetails } from '../../api/posts';
+import {
+  getPostComments,
+  removeComment,
+  creatComment,
+} from '../../api/coments';
+
+export const PostDetails = ({ selectedPostId }) => {
+  const [post, setPost] = useState(null);
+  const [comments, setCommnets] = useState([]);
+  const [visibilCommnet, setVisibilComment] = useState(false);
```

Comment:
name boolean state appropriately

```suggestion
  const [areCommentsVisibile, setAreCommentsVisibile] = useState(false);
```